sanic/blueprints.py

```diff
@@ -146,10 +146,10 @@ def signal(self, event: str, *args, **kwargs):
 
     @staticmethod
     def group(
-        *blueprints,
-        url_prefix="",
-        version=None,
-        strict_slashes=None,
+        *blueprints: Union[Blueprint, BlueprintGroup],
+        url_prefix: Optional[str] = "",
```

Comment:
I am not sure we do accept `None` as a value. I think this should only be `str`.

Reply:
Referenced here. Which one would be more accurate?

https://github.com/sanic-org/sanic/blob/ac784759d56428bcb714eab59caeb92f58391b96/sanic/blueprints.py#L72

Reply:
Hmm... maybe an inconsistency? Let me jump into and see if there is a reason it is `""` for a default there and not `None`.

Reply:
Looks like an inconsistency. The signature in `BlueprintGroup` is correct. Looking at the implementation in `Sanic.blueprint`, a `None` value is safe.

So, let's keep it as `Optional`, but also update the default value to `None`.